drivers/led/Kconfig.pca_common

```diff
@@ -29,4 +29,10 @@ config PCA_COMMON_EXTERNAL_DRIVER
 	help
 	  Set external driver option, configured in dts, on startup.
 
+config PCA_COMMON_DEFAULT_IREF
+	bool "Set default iref parameter on startup"
+	depends on $(dt_compat_enabled,$(DT_COMPAT_NXP_PCA9956))
```

Comment:
Again, is there a reason to allow to enable/disable this if the property is defined in DT ?

Reply:
I have removed it and replaced by `DT_ANY_NODE_HAS_PROP` macro.